tool/deploy.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+main(List args) => grind(args);
+
+@DefaultTask('Deploy software')
+deploy() {
```

Comment:
Yeah, just import the library that defines the task, that should do it. You _may_ have to export the library for the annotation scanner to see that task.